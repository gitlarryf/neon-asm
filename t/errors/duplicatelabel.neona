
PUSHS "Hello, World!"
CALLF output
RET

:output
CALLP print
RET

:output
CALLP str
CALLP print
RET