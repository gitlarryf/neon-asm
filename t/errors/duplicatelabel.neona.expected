Duplicate label definition on line 9.